src/pages/Login/Login.js

```diff
@@ -1,8 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 import './Login.scss';
 
 function Login() {
-  return <div className="login" />;
+  const [inputValue, setInputValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = inputValue;
+
+  const [isActiveEmailBorder, setIsActiveEmailBorder] = useState(false);
+  const [isActivePasswordBorder, setIsActivePasswordBorder] = useState(false);
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    setInputValue({
+      ...inputValue,
+      [name]: value,
+    });
+  };
+
+  const isValidInput = () => {
+    return email.length >= 1 && password.length >= 1;
+  };
```

Comment:
- 함수의 경우엔 동사형으로 이름을 지어주세요! 지금의 경우엔 유효성 검사를 하는 것 이므로, `validateInput` 정도가 적당할 것 같습니다.
- 또한 이 값은 함수로 나타내는 것 보다, 일반 변수로 나타내는 것이 더 어울릴 것 같은 생각입니다.
```suggestion
  const isInputValid = email.length >= 1 && password.length >= 1; // boolean 값
```